R/wb_styles.R

```diff
@@ -79,6 +79,7 @@ import_styles <- function(x) {
 #' @description
 #' Border styles can any of the following: "thin", "thick", "slantDashDot", "none", "mediumDashed", "mediumDashDot", "medium", "hair", "double", "dotted", "dashed", "dashedDotDot", "dashDot"
 #' Border colors are of the following type: c(rgb="FF000000")
```

Comment:
Border colors are of the following type: `wb_color()`

Reply:
That was just a drive by comment :)
`wb_color()` is basically a handler to create the correct xml color type (like theme, tint, rgb)
`all.equal(wb_color("FF000000"), c(rgb = "FF000000"), check.attributes = FALSE)`